### PR TITLE
Add one-time sync Company Data & Company Search docs

### PR DESCRIPTION
**Pull request by jac32**

### CODE REVIEW

File: source/includes/_01-config.html.md [hunk 1]
```diff
@@ -109,9 +109,38 @@ customers have a direct commercial relationship with the data provider.
...
+strings, which should be the [ISO3 country code][wiki-iso3] for the country. At
+least one country must be specified.
+
+## Supported features
+
+The optional `supported_features` field allows your integration to declare 
+support for additional features. The absence of this field from your 
+integration's config implies that no additional features are supported. 
+
+The list of possible features varies depending on the check type and often
```
Comment (jac32) [inline source/includes/_01-config.html.md:123]:
```suggestion
The list of possible features varies depending on the check type but are often used to
```

File: source/includes/_08-additional-features.md.erb [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+        relationship with your integration's data provider, or if they are
+        using the search feature through PassFort's pay-as-you-go reselling scheme.
+      </td>
+    </tr>
+    <tr>
+      <td><code>search_input</code></td>
+      <td>A SearchInput object</td>
+      <td>Yes</td>
+      <td>
+        This field contains the known profile information on the company the
```
Comment (jac32) [inline source/includes/_08-additional-features.md.erb:59]:
```suggestion
        This field contains the known profile information on the company
```

File: source/includes/_08-additional-features.md.erb [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+        company check should be run for.
```
Comment (jac32) [inline source/includes/_08-additional-features.md.erb:60]:
```suggestion
        being searched for.
```

File: source/includes/_08-additional-features.md.erb [hunk 4]
```diff
@@ -0,0 +1,374 @@
+# Additional Features
+
+## Company Search
```
Comment (Diggsey) [inline source/includes/_08-additional-features.md.erb:3]: We should specify the feature identifier here (ie. `COMPANY_SEARCH`)

**Review by Diggsey — CHANGES_REQUESTED** [review]

File: source/json/responses/run_check_sync_company_data.json [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+      "entity_type": "COMPANY",
+      "metadata": {
+        "trade_description": "Business and domestic software development",
+        "tax_ids": [],
+        "structured_company_type": {
+          "ownership_type": "COMPANY",
+          "is_public": false,
+          "is_limited": true
+        },
+        "sic_codes": [
```
Comment (andreeagherman) [inline source/json/responses/run_check_sync_company_data.json:15]: should we remove `sic_codes` if we have `industry_classifications`?

**Review by Diggsey — APPROVED** [review]

**Review by andreeagherman — APPROVED** [review]

**Pull request merged**